From b1c4ee0d0fbc30d634607bf593fad0e864c93ab2 Mon Sep 17 00:00:00 2001
From: Jussi Laako <jussi@sonarnerd.net>
Date: Mon, 26 Oct 2020 20:56:58 +0200
Subject: [PATCH] ALSA: usb-audio: Add Cayin and FiiO VID to DSD detection
 whitelist

Cayin and FiiO DACs have DSD auto-detection support.

Signed-off-by: Jussi Laako <jussi@sonarnerd.net>
---
 sound/usb/quirks.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 5ee9ae1bc642..20ed07a75940 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1738,7 +1738,9 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case 0x25ce:  /* Mytek devices */
 	case 0x278b:  /* Rotel? */
 	case 0x292b:  /* Gustard/Ess based devices */
+	case 0x2972:  /* FiiO devices */
 	case 0x2ab6:  /* T+A devices */
+	case 0x2d87:  /* Cayin? */
 	case 0x3353:  /* Khadas devices */
 	case 0x3842:  /* EVGA */
 	case 0xc502:  /* HiBy devices */
-- 
2.34.1

